lib/usb-device.js

```diff
@@ -16,7 +16,7 @@ class USBDevice {
   }
 
   // opens the device and sets up the endpoints for out object
-  async claimDevice () {
+  claimDevice () {
```

Comment:
I don't know how public this method is, but I would tend to make everything in an exposed API, and most other stuff, `async` even if it doesn't need it. It makes it easier to know how to call the functions, and cost basically nil.